Create Karma config file

alexpattison opened the pull request.

alexpattison committed:
Create Karma config file

jbg46209 commented:
:+1:

cchrispy commented on src/client/services/services.js:
```diff
@@ -92,7 +92,7 @@ app.factory('RestaurantAndRoute', ['$http', function($http) {
 
         //Display an info window when marker is clicked
-        let infoWindow = createInfoWindow(place);
+        let infoWindow = Window(place);
```
is Window a function?

alexpattison replied:
This actually came in from the rebase, not sure why it's in my diff

cchrispy replied:
so it's fine to leave it as Window instead of createInfoWindow?

alexpattison committed:
Add files to config

I'm beginning work on an example test file to be sure I understand the mocks

alexpattison committed:
Implement example to test in browser

I must be missing some dependency. It says app is not defined in controller.js.
